greenplumpython/gpapply.py

```diff
@@ -13,94 +15,59 @@ def randomStringType(stringLength=10):
     letters = string.ascii_lowercase
     return "tmp_type"+''.join(random.choice(letters) for i in range(stringLength))
 
-"""
-list("a", "int4", "b", "int4")
-output list("a", "int8", "b", "float")
-index "col1"
-"""
+def pythonExec(df, funcName, typeName, output_tbl_name, extra_args):
```

Comment:
How to handle extra_args?

Reply:
It's not handled yet, we need another story for this arg